pyccel/errors/errors.py

```diff
@@ -3,6 +3,11 @@
 # go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
 #------------------------------------------------------------------------------------------#
 
+"""
+This module contains classes and methods that manupilate the various errors and warnnigs
```

Comment:
Typos:
`manupilate`->`manipulate`
`warnnigs`->`warnings`

Reply:
after all, i think my fingers are not mine anymore.

Reply:
Haha, time to get a spell checker